test/common.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/../vendor/knative.dev/hack/e2e-tests.sh
+
+function cluster_setup() {
+  header "Building plugin"
+  ${REPO_ROOT_DIR}/hack/build.sh -f || return 1
```

Comment:
Maybe we need to install kn client while cluster is setting up.
Refer to https://github.com/knative/client-contrib/blob/master/test/common.sh#L67